### PR TITLE
Moves schemas to current version.

### PR DESCRIPTION
**Pull request by lgolding**
See https://json-schema.org/understanding-json-schema/reference/schema.html. We are declaring that our schemas conform to the latest version of JSON schema, and do not rely on any non-forward-compatible features of previous drafts (such as "id").

### CODE REVIEW

**Comment by lgolding** [comment]
@michaelcfanning is added to the review. #Closed

**Comment by lgolding** [comment]
@harleenkohli is added to the review. #Closed

File: src/Sarif/Schemata/sarif-2.1.0-rtm.1.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
```
Comment (harleenkohli) [inline src/Sarif/Schemata/sarif-2.1.0-rtm.1.json]: we should update release history!

Comment (lgolding) [reply src/Sarif/Schemata/sarif-2.1.0-rtm.1.json]: Thanks, will do.

---
In reply to: [291363022](https://github.com/microsoft/sarif-sdk/pull/1523#discussion_r291363022) [](ancestors = 291363022)

Comment (lgolding) [reply src/Sarif/Schemata/sarif-2.1.0-rtm.1.json]: Done.

---
In reply to: [291365943](https://github.com/microsoft/sarif-sdk/pull/1523#discussion_r291365943) [](ancestors = 291365943,291363022)

**Pull request merged**